src/lib/__tests__/server-utilities.test.js

```diff
@@ -28,3 +30,45 @@ describe('md5Hash', () => {
     expect(fileNameHash).not.toBe(md5Hash(fileName));
   });
 });
+
+describe('compressJPEG', () => {
+  const filePath = './tmp/test.jpg';
+
+  beforeAll(() => {
+    if (!fs.existsSync('./tmp')) fs.mkdirSync('./tmp');
+
+    // Copy compressible JPEG to `./tmp` directory to simulate JPEG upload
+    return copyFile('./src/lib/__tests__/assets/compression-test.jpg', filePath);
+  });
+
+  /*
+   * We know compression-test.jpg will compress because it is a plain white image
+   * However, not all images will compress depending on the complexity of the images
+   */
+  it('compresses compressible JPEG', () => {
+    expect.assertions(1);
+    const uncompressedSize = fs.statSync(filePath).size;
+    return compressJPEG(filePath).then(() => {
+      const compressedSize = fs.statSync(filePath).size;
+      expect(uncompressedSize).toBeGreaterThan(compressedSize);
+    });
+  });
+
+  /* `./tmp/test.jpg` is already compressed from previous test */
+  it("doesn't recompress already compressed JPEG", () => {
+    expect.assertions(1);
+    const oldCompressedSize = fs.statSync(filePath).size;
+    return compressJPEG(filePath).then(() => {
+      const newCompressedSize = fs.statSync(filePath).size;
+      expect(newCompressedSize).toEqual(oldCompressedSize);
```

Comment:
use `toBe` here. `toBe` is `===` comparison while `toEqual` is a deep comparison. In essence since it's just a number it won't make a difference but just for semantics, `toEqual` is used for deep comparisons of objects.